doc/api/deprecations.md

```diff
@@ -1364,7 +1364,7 @@ The `NODE_REPL_MODE` environment variable is used to set the underlying
 removed. Please use `sloppy` instead.
 
 <a id="DEP0066"></a>
-### DEP0066: outgoingMessage.\_headers, outgoingMessage.\_headerNames
+### DEP0066: outgoingMessage.prototype.\_headers, outgoingMessage.prototype.\_headerNames
```

Comment:
I think there is nothing wrong with the original. `outgoingMessage` refers to an instance of `OutgoingMessage` so `outgoingMessage._headers` is ok.

Reply:
@lpinca 

compare https://nodejs.org/dist/latest-v12.x/docs/api/deprecations.html#deprecations_dep0066_outgoingmessage_headers_outgoingmessage_headernames with https://nodejs.org/dist/latest-v12.x/docs/api/deprecations.html#deprecations_dep0067_outgoingmessage_prototype_renderheaders

as well as the deprecation message itself:

https://github.com/nodejs/node/blob/908292cf1f551c614a733d858528ffb13fb3a524/lib/_http_outgoing.js#L113

I think that consistency is key as well as finding the doc with a search engine (SEO).

Reply:
Oh ok, so it's for consistency.

Reply:
In that case can you please capitalize `outgoingMessage`? It should be `OutgoingMessage`.

Reply:
ah, good catch! I'll fix that as well ...

Reply:
just noticed another inconsistency: also added prototype to the suggested methods, compare to: https://nodejs.org/dist/latest-v12.x/docs/api/deprecations.html#deprecations_dep0001_http_outgoingmessage_prototype_flush

which also mentions the module name: `http.OutgoingMessage` :/